docs/app.yaml

```diff
@@ -1,5 +1,5 @@
-# application: simpegdocs
-# version: 1
+# application: simpeg-dev-docs
```

Comment:
ah, no I was testing deployment locally and wanted to do that on dev-docs. Would you prefer I update this back to simpegdoc before merging?